go/client/cmd_team_list_memberships.go

```diff
@@ -248,7 +248,7 @@ func (c *CmdTeamListMemberships) outputRole(role string, members []keybase1.Team
 
 func (c *CmdTeamListMemberships) outputInvites(invites map[keybase1.TeamInviteID]keybase1.AnnotatedTeamInvite) {
 	for _, invite := range invites {
-		fmtstring := "%s\t%s*\t%s\t(* invited by %s; awaiting acceptance)\n"
+		fmtstring := "%s\t%s*\t%s\t(* added by %s; awaiting acceptance)\n"
```

Comment:
did we have other issues around this display? @zapu  might have said something?

Reply:
The keybase users won't even show up here once I push the next commit...

Reply:
I think my issue was that we were showing old uid%seqno pairs for users that has reset and there was no way to remove such invite, but that was fixed in `go/teams/*` code in https://github.com/keybase/client/pull/8450